src/store/modules/facility/actions.ts

```diff
@@ -146,6 +146,46 @@ const actions: ActionTree<FacilityState, RootState> = {
     commit(types.FACILITY_CURRENT_UPDATED, facility)
   },
 
+  async fetchCreatedFacility({ commit, state }, payload) {
```

Comment:
There should already be some action to fetch facility details, as we already fetching facility details on the detail page.